index.html

```diff
@@ -67,7 +67,8 @@ <h2>Childcare</h2>
       <section id="location">
         <h2>Location</h2>
 
-        We're still looking for a venue. If you have any suggestions, please let us know!
+        We're aiming to have the 2023 Ruby Unconf at the <a href="https://www.haw-hamburg.de/">HAW Hamburg</a> again.
+        (<a href="https://www.openstreetmap.org/way/23391525#map=18/53.55653/10.02314">Berliner Tor 5, 20099 Hamburg, Germany</a>)
```

Comment:
proposal: open sites in new windows

```suggestion
        We're aiming to have the 2023 Ruby Unconf at the <a href="https://www.haw-hamburg.de/" target="_blank">HAW Hamburg</a> again.
        (<a href="https://www.openstreetmap.org/way/23391525#map=18/53.55653/10.02314" target="_blank">Berliner Tor 5, 20099 Hamburg, Germany</a>)
```